radixdlt-core/radixdlt/src/integration/java/com/radixdlt/integration/mempool/MempoolRelayTest.java

```diff
@@ -265,7 +265,6 @@ public void full_node_should_relay_mempool_messages_so_they_can_be_processed_by_
 		// assert that validators have an empty mempool, but not the full nodes
 		this.validators.forEach(n -> assertEquals(0L, getCounter(n, CounterType.MEMPOOL_COUNT)));
 		this.fullNodes.forEach(n -> assertTrue(getCounter(n, CounterType.MEMPOOL_COUNT) >= 1L));
-
```

Comment:
Please, roll back this file, there are no changes in the code

Reply:
👍